testsuite/forge/src/backend/k8s/cluster_helper.rs

```diff
@@ -414,41 +402,35 @@ pub async fn install_testnet_resources(
     let new_era = generate_new_era();
 
     // get forge override helm values and cache it
-    let aptos_node_forge_helm_values_yaml = format!(
-        include_str!(APTOS_NODE_FORGE_HELM_VALUES!()),
-        num_validators = num_validators,
-        num_fullnodes = num_fullnodes,
-        era = &new_era,
-        image_tag = &node_image_tag,
-        enable_haproxy = enable_haproxy,
-        namespace = &kube_namespace,
-    );
+
+    let aptos_node_forge_helm_values_yaml = construct_node_helm_values(
+        node_helm_config_fn,
+        fs::read_to_string(
+            "testsuite/forge/src/backend/k8s/helm-values/aptos-node-default-values.yaml",
```

Comment:
file path probably needs to be just `helm-values/aptos-node-default-values.yaml` to be consistent with the docker build

Reply:
Yes, but it fails to find this file in local mode as the test is running from the repository root. Do you have any suggestions on how to deal with that? Earlier, it was not an issue because the `include_str!` was being done at compile time and it was able to find it.

Reply:
I can make a special case for `local mode` but its probably too ugly :(